src/autopas/tuning/AutoTuner.cpp

```diff
@@ -103,8 +107,11 @@ bool AutoTuner::tuneConfiguration() {
   };
 
   // Determine where in a tuning phase we are
-  if (_iterationsSinceTuning == _tuningInterval) {
-    // CASE: Start of a tuning phase
+  if (_iterationsSinceTuning == _tuningInterval and not(_lastTuningIterations >= _tuningInterval)) {
+    // CASE: Start of a tuning phase (if _lastTuningIterations > _tuningInterval the current tuning phase takes more
+    // iterations tahn the tuning interval -> continue tuning)
```

Comment:
```suggestion
  // if _lastTuningIterations >= _tuningInterval the current tuning phase takes more iterations than the tuning interval
  // -> continue tuning
  if (_iterationsSinceTuning == _tuningInterval and not(_lastTuningIterations >= _tuningInterval)) {
    // CASE: Start of a tuning phase
```
Just some reformatting

Reply:
done in [b9e4111](https://github.com/AutoPas/AutoPas/pull/943/commits/b9e4111c93ce42af332b045755214316b102b040)